db/migrate/20200310140717_create_family.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFamily < ActiveRecord::Migration[6.0]
+  def change
+    create_table :families do |t|
+      t.timestamps
+    end
+
+    add_reference :users, :family, null: true, foreign_key: true
+    add_column :users, :member_type, :integer
```

Comment:
Well, that is not exactly what i meant. I just don't know how to explain clearly.
Unless your migrations are not merged - you can contain all changes in one migration, but which is related to one table.
For example:
migration 1 - you have created table
migration 2 - you have added one more field
migration 3 - you have removed field
migration 4 - you created another table
So, unless these migration are merged, you can merge 1, 2 and 3 migrations in 1(create table). As a result you will push two migrations, migration 1 and migration 4. I hope that make sense.

As for you current migration - let it be.